### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'length')

### ISSUE THREAD

**Issue by michael-land** [vercel/next.js#50388]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

```js
/** @type {import('next').NextConfig} */
export default {
  experimental: {
    appDir: true,
    serverActions: true,
  },
};
```

```bash
- error uncaughtException: TypeError: Cannot read properties of undefined (reading 'length')
    at WasmHash._updateWithBuffer (/Users/./hubspot-kit/monorepo3/node_modules/.pnpm/next@13.4.5-canary.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/compiled/webpack/bundle5.js:28:1352524)
    at WasmHash.update (/Users/./hubspot-kit/monorepo3/node_modules/.pnpm/next@13.4.5-canary.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/compiled/webpack/bundle5.js:28:1351859)
    at BatchedHash.update (/Users/./hubspot-kit/monorepo3/node_modules/.pnpm/next@13.4.5-canary.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/compiled/webpack/bundle5.js:28:1348216)
    at /Users/./hubspot-kit/monorepo3/node_modules/.pnpm/next@13.4.5-canary.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/compiled/webpack/bundle5.js:28:292480
    at processQueue (/Users/./hubspot-kit/monorepo3/node_modules/.pnpm/next@13.4.5-canary.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/compiled/webpack/bundle5.js:28:1365665)
    at process.processTicksAndRejections (node:internal/process/task_queues:77:11)
```

After updating to version 13.4.4, the "next dev" no longer functions. Has anyone else experienced this issue, or is it just me?

**Comment by marozzocom** [comment]
I can confirm having same issue on 13.4.4 and 13.4.5-canary.0. Reverting to 13.4.3 removed the error.

**Comment by balazsorban44** [comment]
Closing this issue. Please open a new one with the reproduction steps and don't ignore the issue template. 🙏 this will let us investigate the issue much faster and will result in a faster resolution for you too. Thanks!

**Comment by Torsten85** [comment]
I had the same error.. as it turns out I had a misconfiguration in my `tailwind.config.cjs`! 

I'm also working in a pnpm monorepo and I need tailwind to include a specific dependency as content, so I added `./node_modules/my-other-package/**/*.{ts,tsx}` to the configuration. But that also included every dependency of my-other-package (`./node_modules/my-package/node_modules/**`).  And those dependency are links to the pnpm store... I guess at some point, the bundle simply gave up 😉 

Long story short, changing the content in my tailwindconfig to `./node_modules/my-other-package/src/**/*{.ts,tsx}` did the trick for me.

**Comment by stevejcox** [comment]
We're seeing the WasmHash error back in 13.4.4

**Comment by awilderink** [comment]
> I had the same error.. as it turns out I had a misconfiguration in my `tailwind.config.cjs`!
> 
> I'm also working in a pnpm monorepo and I need tailwind to include a specific dependency as content, so I added `./node_modules/my-other-package/**/*.{ts,tsx}` to the configuration. But that also included every dependency of my-other-package (`./node_modules/my-package/node_modules/**`). And those dependency are links to the pnpm store... I guess at some point, the bundle simply gave up 😉
> 
> Long story short, changing the content in my tailwindconfig to `./node_modules/my-other-package/src/**/*{.ts,tsx}` did the trick for me.

Live saver, exactly the same setup for me and this fixed it. Weird it didn't occur before updating to 13.4.4
